### PR TITLE
Add an LHE-based mc relval with multiple lumi sections and concurrent lumi settings

### PR DESCRIPTION
**Pull request by fabiocos**
#### PR description:

This PR adds an LHE-based simulation relval workflow where 2 lumi sections are produced, and the concurrent luminosity block settings are activated. This is a first attempt to address the request by @Dr15Jones discussed in #26348 , and test the lumi section boundary crossing that is usual in production and that sometimes shows problems (see for instance #25708 ).

In order to test the boundary crossing from LHE_based input onwards, the workflow 1361.18 is modified by forcing the EmptySource to have 5 events per lumi section, and therefore forcing the standard 10 events sample to contain 2 lumi sections.

#### PR validation:

If ```--nThreads 4``` the new workflow 1361.181 has the new correct settings, for instance in step2 (DIGI):

```
%MSG-i ThreadStreamSetup:  (NoModuleName) 04-Sep-2019 16:12:47 CEST pre-events
setting # threads 4
setting # streams 4
%MSG
04-Sep-2019 16:12:53 CEST  Initiating request to open file file:step1.root
04-Sep-2019 16:12:54 CEST  Successfully opened file file:step1.root
%MSG-s ModulesSynchingOnLumis:  AfterModConstruction 04-Sep-2019 16:12:59 CEST  pre-events
The following modules require synchronizing on LuminosityBlock boundaries:
  HLTrigReport hltTrigReport
%MSG
Begin processing the 1st record. Run 1, Event 2, LumiSection 1 on stream 3 at 04-Sep-2019 16:13:05.814 CEST
Begin processing the 2nd record. Run 1, Event 3, LumiSection 1 on stream 0 at 04-Sep-2019 16:13:05.814 CEST
Begin processing the 3rd record. Run 1, Event 4, LumiSection 1 on stream 2 at 04-Sep-2019 16:13:05.814 CEST
Begin processing the 4th record. Run 1, Event 5, LumiSection 1 on stream 1 at 04-Sep-2019 16:13:05.814 CEST
DTTSPhi::addTracoT: preview not present in TRACO trigger or its code=0  trigger not added!
Begin processing the 5th record. Run 1, Event 1, LumiSection 1 on stream 1 at 04-Sep-2019 16:14:14.785 CEST
Begin processing the 6th record. Run 1, Event 9, LumiSection 2 on stream 3 at 04-Sep-2019 16:14:16.953 CEST
Begin processing the 7th record. Run 1, Event 7, LumiSection 2 on stream 0 at 04-Sep-2019 16:14:16.953 CEST
Begin processing the 8th record. Run 1, Event 6, LumiSection 2 on stream 2 at 04-Sep-2019 16:14:16.953 CEST
Begin processing the 9th record. Run 1, Event 8, LumiSection 2 on stream 1 at 04-Sep-2019 16:14:16.954 CEST
Begin processing the 10th record. Run 1, Event 10, LumiSection 2 on stream 0 at 04-Sep-2019 16:14:18.686 CEST
04-Sep-2019 16:14:20 CEST  Closed file file:step1.root
```

### CODE REVIEW

**Comment by fabiocos** [comment]
please test workflow 1631.181

**Comment by chayanit** [comment]
please test workflow 1361.181

**Comment by fabiocos** [comment]
@prebello @zhenhu @chayanit @Dr15Jones @kpedro88 comments?

**Comment by fabiocos** [comment]
for the time being I think we can have a dedicated workflow with these settings, then we may consider whether to extend them to a larger set. Actually the ```nConcurrentLumis``` and ```nStreams``` settings are active only of ```nThreads``` is greater than 1, which is never the case in PR tests (but it is in IB), so even a global change could be activated in case, as the new test wf looks to me the only one spanning more lumi sections at the moment.

@srimanob FYI

**Comment by Dr15Jones** [comment]
@fabiocos I'm all for this. At least one of the jobs doesn't actually have efficient concurrent lumis since it had the warning
```
%MSG-s ModulesSynchingOnLumis:  AfterModConstruction 04-Sep-2019 16:12:59 CEST  pre-events
The following modules require synchronizing on LuminosityBlock boundaries:
  HLTrigReport hltTrigReport
%MSG
```

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -1491,6 +1491,19 @@ def lhegensim2018(fragment,howMuch):
...
 
+# Workflows for multiple concurrent lumi blocks
+
+step1LHEGenSimUp2018MultiLumi = merge ([{'--customise_commands': '"process.source.numberEventsInLuminosityBlock=cms.untracked.uint32(5)"',
+                                         '--nStreams': 4,
+                                         '--nConcurrentLumis': 2,
+                                         },step1LHEGenSimUp2018Default])
+
+def lhegensim2018ml(fragment,howMuch):
+    global step1LHEGenSimUp2018MultiLumi
```
Comment (kpedro88) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: rather than declaring this variable global, I think it could just be defined inside this function

Comment (fabiocos) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: @kpedro88 if we want to restrict its use to this specific function I agree, probably it makes sense to declare a global variable for the ```nConcurrentLumis``` and ```nStreams``` definition (which can be used also for data) and move the rest inside the function as you suggest

**Comment by fabiocos** [comment]
I understand that @srimanob would possibly like to make an alternative proposal, so before moving forward with this PR I would like to hear from him as well

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 2]
```diff
@@ -625,6 +625,9 @@
 workflows[1362.18] = ['', ['VBFHToZZTo4Nu_M125_Pow_py8_Evt_13UP18','DIGIUP18','RECOUP18','HARVESTUP18']]
 workflows[1363.18] = ['', ['VBFHToBB_M125_Pow_py8_Evt_13UP18','DIGIUP18','RECOUP18','HARVESTUP18']]
 
+#2018 workflows starting from gridpacks LHE generation with multiple concurrent lumi sections
+workflows[1361.181] = ['', ['GluGluHToZZTo4L_M125_Pow_py8_Evt_13UP18ml','DIGIUP18ml','RECOUP18ml','HARVESTUP18']]
+
 # fullSim 13TeV normal workflows starting from pLHE
 workflows[1370] = ['', ['GluGluHToGG_M125_Pow_MINLO_NNLOPS_py8_13','Hadronizer_TuneCUETP8M1_13TeV_powhegEmissionVeto_2p_HToGG_M125_13','DIGIUP15','RECOUP15','HARVESTUP15']]
```
Comment (chayanit) [inline Configuration/PyReleaseValidation/python/relval_standard.py]: Hi @fabiocos , i'm not sure if understood correctly but from https://github.com/cms-sw/cmssw/issues/25708 @srimanob concerned about pLHE+GS production. We have workflows 1370,1370.17,1370.18 in our standard relval list which start from external source LHE (or pLHE). The one you added will produce LHE first through gridpack and then get into GS step, similar to our wmLHEGS in production. Do we plan to expand these settings in relval standard list? If so, i would suggest to work on top of 1370,1370.17,1370.18.

Comment (fabiocos) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: @chayanit my reference to #25708 was an example of issue that happens at the boundary between lumi sections. This is a minimal proposal to add 1 wf that tests systematically the crossing of such a boundary in the whole chain. This can be expanded of course, but there is no big advantage in my view to add just clones of this for older setups of this workflow.

**Comment by chayanit** [comment]
+1

**Comment by fabiocos** [comment]
@kpedro88 any further comment/suggestion?

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+1

**Comment by Dr15Jones** [comment]
Looking at the logs from the new workflow, particular step3, we can see that all the DQM modules are prohibiting the framework from begin abe to use concurrent LuminosityBlocks.

Excerpt from the log
```
%MSG-s ModulesSynchingOnLumis:  AfterModConstruction 10-Sep-2019 14:43:47 CEST  pre-events
The following modules require synchronizing on LuminosityBlock boundaries:
  AlcaBeamMonitor AlcaBeamMonitor
  CaloTowersAnalyzer AllCaloTowersDQMOffline
  CaloTowersValidation AllCaloTowersValidation
  HcalDigisValidation AllHcalDigisValidation
  SimHitsValidationHcal AllSimHitsValidation
  SUSYDQMAnalyzer AnalyzeSUSYDQM
  B2GDQM B2GDQM
  BDHadronTrackMonitoringAnalyzer BDHadronTrackMonitoringAnalyze
  LogMessageMonitor ClusterizerLogMessageMonCommon
  DQMMessageLogger DQMMessageLogger
  DTDataIntegrityTask DTDataIntegrityTask
  TopDiLeptonOfflineDQM DiElectronDQM
  TopDiLeptonOfflineDQM DiMuonDQM
  TopDiLeptonOfflineDQM ElecMuonDQM
  PFCandidateDQMAnalyzer ElectronValidation1
  ExoticaDQM ExoticaDQM
  FSQDQM FSQDQM
  HcalSimHitsValidation HcalSimHitsAnalyser
  HiggsDQM HiggsDQM
  BPhysicsValidation JPsiMuMuValidation
  JetTester JetAnalyzerAk4Calo
  JetTester JetAnalyzerAk4PF
  JetTester JetAnalyzerAk4PFCHS
  JetTester JetAnalyzerAk4PFCHSMiniAOD
  JetTester JetAnalyzerAk4PFPUPPIMiniAOD
  JetTester JetAnalyzerAk8PFPUPPIMiniAOD
  PFJetDQMAnalyzer JetResValidation1
  PFJetDQMAnalyzer JetResValidation2
  PFJetDQMAnalyzer JetValidation1
  PFJetDQMAnalyzer JetValidation2
  V0Monitor KshortMonitoring
  V0Monitor KshortWlxy16Monitoring
  L1Validator L1Validator
  V0Monitor LambdaMonitoring
  BPhysicsSpectrum LambdaSpectrum
  V0Monitor LambdaWlxy16Monitoring
  BPhysicsValidation LambdabPiPiMuMuValidation
  LogMessageMonitor LocalRecoLogMessageMonCommon
  EfficiencyAnalyzer LooseMuonEfficiencyAnalyzer
  EfficiencyAnalyzer LooseMuonEfficiencyAnalyzer_miniAOD
  PFMETDQMAnalyzer METValidation1
  PFMETDQMAnalyzer METValidation2
  EfficiencyAnalyzer MediumMuonEfficiencyAnalyzer
  EfficiencyAnalyzer MediumMuonEfficiencyAnalyzer_miniAOD
  TrackingMonitor MonitorTrackGLBMuons
  TrackingMonitor MonitorTrackMuonsInnerTrack
  MonitorTrackResiduals MonitorTrackResiduals
  TrackingMonitor MonitorTrackSTAMuons
  MuonIsolationDQM MuIsoDQM_glb
  MuonIsolationDQM MuIsoDQM_glb_miniAOD
  MuonIsolationDQM MuIsoDQM_sta
  MuonIsolationDQM MuIsoDQM_sta_miniAOD
  MuonIsolationDQM MuIsoDQM_trk
  MuonIsolationDQM MuIsoDQM_trk_miniAOD
  MuIsoValidation MuIsoValidation_global
  MuIsoValidation MuIsoValidation_inc
  MuonMiniAOD MuonMiniAOD
  HcalNoiseRates NoiseRatesDQMOffline
  NoiseRates NoiseRatesValidation
  BPhysicsSpectrum PsiSpectrum
  HcalRecHitsAnalyzer RecHitsDQMOffline
  HcalRecHitsValidation RecHitsValidation
  TauTagValidation RunHPSValidationQCD
  TauTagValidation RunHPSValidationRealData
  TauTagValidation RunHPSValidationRealElectronsData
  TauTagValidation RunHPSValidationRealMuonsData
  TauTagValidation RunHPSValidationZEE
  TauTagValidation RunHPSValidationZMM
  TauTagValidation RunHPSValidationZTT
  LogMessageMonitor SeedingLogMessageMonCommon
  SiPixelPhase1Clusters SiPixelPhase1ClustersAnalyzer
  SiPixelPhase1DeadFEDChannels SiPixelPhase1DeadFEDChannelsAnalyzer
  SiPixelPhase1Digis SiPixelPhase1DigisAnalyzer
  SiPixelPhase1DigisV SiPixelPhase1DigisAnalyzerV
  SiPixelPhase1HitsV SiPixelPhase1HitsAnalyzerV
  SiPixelPhase1RawData SiPixelPhase1RawDataAnalyzer
  SiPixelPhase1RecHits SiPixelPhase1RecHitsAnalyzer
  SiPixelPhase1RecHitsV SiPixelPhase1RecHitsAnalyzerV
  SiPixelPhase1TrackClusters SiPixelPhase1TrackClustersAnalyzer
  SiPixelPhase1TrackClustersV SiPixelPhase1TrackClustersAnalyzerV
  SiPixelPhase1TrackEfficiency SiPixelPhase1TrackEfficiencyAnalyzer
  SiPixelPhase1TrackResiduals SiPixelPhase1TrackResidualsAnalyzer
  SiPixelPhase1TrackingParticleV SiPixelPhase1TrackingParticleAnalyzerV
  SiStripMonitorCluster SiStripMonitorClusterBPTX
  SiStripMonitorDigi SiStripMonitorDigi
  SiStripMonitorTrack SiStripMonitorTrackCommon
  SiStripMonitorTrack SiStripMonitorTrackIB
  SiStripTrackingRecHitsValid StripTrackingRecHitsValid
  TTbarSpinCorrHepMCAnalyzer TTbarAnalyzeSpinCorr
  EfficiencyAnalyzer TightMuonEfficiencyAnalyzer
  EfficiencyAnalyzer TightMuonEfficiencyAnalyzer_miniAOD
  LogMessageMonitor TrackCandidateLogMessageMonCommon
  LogMessageMonitor TrackFinderLogMessageMonCommon
  TrackEfficiencyMonitor TrackMon_ckf
  TrackingMonitor TrackSeedMondetachedQuadStep
  TrackingMonitor TrackSeedMondetachedTripletStep
  TrackingMonitor TrackSeedMonhighPtTripletStep
  TrackingMonitor TrackSeedMoninitialStep
  TrackingMonitor TrackSeedMonjetCoreRegionalStep
  TrackingMonitor TrackSeedMonlowPtQuadStep
  TrackingMonitor TrackSeedMonlowPtTripletStep
  TrackingMonitor TrackSeedMonmixedTripletStep
  TrackingMonitor TrackSeedMonmuonSeededStepInOut
  TrackingMonitor TrackSeedMonmuonSeededStepOutIn
  TrackingMonitor TrackSeedMonpixelLessStep
  TrackingMonitor TrackSeedMonpixelPairStep
  TrackingMonitor TrackSeedMontobTecStep
  TrackSplittingMonitor TrackSplitMonitor
  TrackingMonitor TrackerCollisionSelectedTrackMonCommongeneralTracks
  TrackingMonitor TrackerCollisionSelectedTrackMonCommonhighPurityPV0p1
  TrackingMonitor TrackerCollisionSelectedTrackMonCommonhighPurityPt1
  TrackingMonitor TrackerCollisionSelectedTrackMonCommonhighPurityPtRange0to1
  HiggsValidation Z0testValidation
  TTbar_GenLepAnalyzer analyzeGenElecs
  TTbar_GenJetAnalyzer analyzeGenJets
  TTbar_GenLepAnalyzer analyzeGenMuons
  TTbar_GenLepAnalyzer analyzeGenNtrns
  TTbar_Kinematics analyzeTopKinematics
  BTagPerformanceAnalyzerOnData bTagAnalysis
  BTagPerformanceAnalyzerMC bTagValidationNoall
  BasicGenParticleValidation basicGenParticleValidation
  BasicHepMCValidation basicHepMCValidation
  BPhysicsOniaDQM bphysicsOniaDQM
  METAnalyzer caloMetDQMAnalyzer
  CastorMonitorModule castorOfflineMonitor
  CSCDigiValidation cscDigiValidation
  CSCOfflineMonitor cscMonitor
  CTPPSPixelDQMSource ctppsPixelDQMSource
  dEdxHitAnalyzer dEdxHitMonCommon
  dEdxAnalyzer dEdxMonCommon
  DiMuonHistograms diMuonHistos
  DiMuonHistograms diMuonHistos_miniAOD
  DigiPhase1Task digiPhase1Task
  DigiTask digiTask
  MuonTrackValidator displacedGlbMuonTrackVMuonAssoc
  MuonTrackValidator displacedStaMuonTrackVMuonAssoc
  MuonTrackValidator displacedStaSeedTrackVMuonAssoc
  MuonTrackValidator displacedTrackVMuonAssoc
  CSCMonitorModule dqmCSCClient
  DQMDcsInfo dqmDcsInfo
  ElectronAnalyzer dqmElectronAnalysisAllElectrons
  ElectronAnalyzer dqmElectronAnalysisSelectionEt
  ElectronAnalyzer dqmElectronAnalysisSelectionEtIso
  ElectronGeneralAnalyzer dqmElectronGeneralAnalysis
  ElectronTagProbeAnalyzer dqmElectronTagProbeAnalysis
  DQMEventInfo dqmEnvL1T
  DQMEventInfo dqmEnvL1TEMU
  DQMEventInfo dqmInfoDT
  DQMEventInfo dqmInfoEcal
  DQMEventInfo dqmInfoMuons
  DQMEventInfo dqmInfoSiStrip
  DQMEventInfo dqmInfoTauV
  DQMEventInfo dqmInfoTracking
  DrellYanValidation drellYanEleValidation
  DrellYanValidation drellYanMuoValidation
  DTDCSByLumiTask dtDCSByLumiMonitor
  DTChamberEfficiency dtEfficiencyMonitor
  DTResolutionAnalysisTask dtResolutionAnalysisMonitor
  DTRunConditionVar dtRunConditionVar
  DTSegmentAnalysisTask dtSegmentAnalysisMonitor
  DTSegmentsTask dtSegmentsMonitor
  DTTriggerEfficiencyTask dtTriggerEfficiencyMonitor
  EcalBarrelDigisValidation ecalBarrelDigisValidation
  EcalBarrelRecHitsValidation ecalBarrelRecHitsValidation
  EcalDigisValidation ecalDigisValidation
  EcalEndcapDigisValidation ecalEndcapDigisValidation
  EcalEndcapRecHitsValidation ecalEndcapRecHitsValidation
  EcalFEDMonitor ecalFEDMonitor
  EcalDQMonitorTask ecalMonitorTask
  EcalPileUpDepMonitor ecalPileUpDepMonitor
  EcalPreshowerDigisValidation ecalPreshowerDigisValidation
  ESFEDIntegrityTask ecalPreshowerFEDIntegrityTask
  ESIntegrityTask ecalPreshowerIntegrityTask
  ESOccupancyTask ecalPreshowerOccupancyTask
  ESRawDataTask ecalPreshowerRawDataTask
  EcalPreshowerRecHitsValidation ecalPreshowerRecHitsValidation
  ESRecoSummary ecalPreshowerRecoSummary
  ESTrendTask ecalPreshowerTrendTask
  EcalRecHitsValidation ecalRecHitsValidation
  EcalSelectiveReadoutValidation ecalSelectiveReadoutValidation
  EcalZmassTask ecalzmasstask
  EgammaBasicClusters egammaBasicClusterAnalyzer
  EgammaSuperClusters egammaSuperClusterAnalyzer
  ElectronMcFakeValidator electronMcFakeValidator
  ElectronMcSignalValidator electronMcSignalValidator
  ElectronMcSignalValidatorMiniAOD electronMcSignalValidatorMiniAOD
  ElectronMcSignalValidator electronMcSignalValidatorPt1000
  GEMDigiTrackMatch gemDigiTrackValidation
  GEMCheckGeometry gemGeometryChecker
  GEMRecHitTrackMatch gemRecHitTrackValidation
  GEMRecHitsValidation gemRecHitsValidation
  GEMHitsValidation gemSimHitValidation
  GEMSimTrackMatch gemSimTrackValidation
  GEMStripDigiValidation gemStripValidation
  METTester genMetTrueAnalyzer
  PFCandidateDQMAnalyzer genParticlesValidation
  SegmentTrackAnalyzer glbMuonSegmentAnalyzer
  MuonTrackValidator glbMuonTrackVMuonAssoc
  GlobalDigisAnalyzer globaldigisanalyze
  GlobalHitsAnalyzer globalhitsanalyze
  GlobalRecHitsAnalyzer globalrechitsanalyze
  HcalSimHitStudy hcalSimHitStudy
  HiggsValidation hplusValidation
  JetAnalyzer jetDQMAnalyzerAk4CaloCleaned
  JetAnalyzer jetDQMAnalyzerAk4PFCHSCleaned
  JetAnalyzer jetDQMAnalyzerAk4PFCHSCleanedMiniAOD
  JetAnalyzer jetDQMAnalyzerAk4PFCHSPuppiCleanedMiniAOD
  JetAnalyzer jetDQMAnalyzerAk4PFCHSUncleanedMiniAOD
  JetAnalyzer jetDQMAnalyzerAk4PFCleaned
  JetAnalyzer jetDQMAnalyzerAk4PFUncleaned
  JetAnalyzer jetDQMAnalyzerAk8PFPUPPICleanedMiniAOD
  L1TObjectsTiming l1tObjectsTiming
  L1TStage2BMTF l1tStage2Bmtf
  L1TStage2RegionalMuonCandComp l1tStage2BmtfOutVsuGMTIn
  L1TStage2BMTF l1tStage2BmtfSecond
  L1TMP7ZeroSupp l1tStage2BmtfZeroSupp
  L1TStage2CaloLayer1 l1tStage2CaloLayer1
  L1TStage2CaloLayer2 l1tStage2CaloLayer2
  L1TStage2CaloLayer2 l1tStage2CaloLayer2EmulOffline
  L1TStage2EMTF l1tStage2Emtf
  L1TStage2RegionalMuonCandComp l1tStage2EmtfOutVsuGMTIn
  L1TStage2OMTF l1tStage2Omtf
  L1TStage2RegionalMuonCandComp l1tStage2OmtfOutVsuGMTIn
  L1TStage2uGMT l1tStage2uGMT
  L1TStage2uGMT l1tStage2uGMTEmul
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateBMTF
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateBMTFEmul
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateEMTFNeg
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateEMTFNegEmul
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateEMTFPos
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateEMTFPosEmul
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateOMTFNeg
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateOMTFNegEmul
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateOMTFPos
  L1TStage2uGMTMuon l1tStage2uGMTIntermediateOMTFPosEmul
  L1TStage2MuonComp l1tStage2uGMTOutVsuGTIn
  L1TMP7ZeroSupp l1tStage2uGMTZeroSupp
  L1TStage2uGTCaloLayer2Comp l1tStage2uGTCaloLayer2Comp
  L1TStage2RegionalMuonCandComp l1tdeStage2Bmtf
  L1TStage2RegionalMuonCandComp l1tdeStage2BmtfSecond
  L1TdeStage2CaloLayer2 l1tdeStage2CaloLayer2Offline
  L1TdeStage2EMTF l1tdeStage2Emtf
  L1TStage2RegionalMuonCandComp l1tdeStage2EmtfComp
  L1TStage2RegionalMuonCandComp l1tdeStage2Omtf
  L1TStage2MuonComp l1tdeStage2uGMT
  L1TStage2MuonComp l1tdeStage2uGMTIntermediateBMTF
  L1TStage2MuonComp l1tdeStage2uGMTIntermediateEMTFNeg
  L1TStage2MuonComp l1tdeStage2uGMTIntermediateEMTFPos
  L1TStage2MuonComp l1tdeStage2uGMTIntermediateOMTFNeg
  L1TStage2MuonComp l1tdeStage2uGMTIntermediateOMTFPos
  L1TdeStage2uGT l1tdeStage2uGT
  TrackingRecoMaterialAnalyser materialDumperAnalyzer
  MBUEandQCDValidation mbueAndqcdValidation
  METTester metAnalyzer
  MixCollectionValidation mixCollectionValidation
  MuonEnergyDepositAnalyzer muonEnergyDepositAnalyzer
  MuonIdDQM muonIdDQM
  MuonIdDQM muonIdDQMInVal
  MuonIdVal muonIdVal
  MuonKinVsEtaAnalyzer muonKinVsEtaAnalyzer
  MuonKinVsEtaAnalyzer muonKinVsEtaAnalyzer_miniAOD
  MuonPFAnalyzer muonPFsequence
  MuonPFAnalyzer muonPFsequenceMC
  MuonRecoAnalyzer muonRecoAnalyzer
  MuonRecoAnalyzer muonRecoAnalyzer_miniAOD
  MuonRecoOneHLT muonRecoOneHLT
  MuonSeedsAnalyzer muonSeedsAnalyzer
  MuonDTDigis muondtdigianalyzer
  OffsetAnalyzerDQM offsetAnalyzerDQM
  DQMPFCandidateAnalyzer packedCandidateDQMAnalyzerMiniAOD
  PackedCandidateTrackValidator packedCandidateTrackValidator
  PackedCandidateTrackValidator packedCandidateTrackValidatorLostTracks
  DQMPFCandidateAnalyzer pfCandidateDQMAnalyzer
  METAnalyzer pfChMetDQMAnalyzer
  PFCandidateDQMAnalyzer pfElectronValidation1
  PFJetAnalyzerDQM pfJetAnalyzerDQM
  PFJetDQMAnalyzer pfJetResValidation1
  PFJetDQMAnalyzer pfJetValidation1
  PFJetDQMAnalyzer pfJetValidation2
  PFMETDQMAnalyzer pfMETValidation1
  PFMETDQMAnalyzer pfMETValidation2
  METTester pfMetAnalyzer
  METAnalyzer pfMetDQMAnalyzer
  METAnalyzer pfMetDQMAnalyzerMiniAOD
  METAnalyzer pfMetT1DQMAnalyzer
  PhotonValidator pfPhotonValidation
  METTester pfPuppiMetAnalyzerMiniAOD
  METAnalyzer pfPuppiMetDQMAnalyzerMiniAOD
  METTester pfType01CorrectedMetAnalyzer
  METTester pfType0CorrectedMetAnalyzer
  METTester pfType1CorrectedMetAnalyzer
  METTester pfType1CorrectedMetAnalyzerMiniAOD
  PFClusterValidation pfclusterAnalyzer
  PhotonAnalyzer photonAnalysis
  PhotonValidator photonValidation
  PhotonValidatorMiniAOD photonValidationMiniAOD
  PiZeroAnalyzer piZeroAnalysis
  PrimaryVertexResolution primaryVertexResolution
  PrimaryVertexMonitor pvMonitor
  QcdPhotonsDQM qcdPhotonsDQM
  RecHitTask recHitTask
  RecoMuonValidator recoMuonVMuAssoc_glb
  RecoMuonValidator recoMuonVMuAssoc_sta
  RecoMuonValidator recoMuonVMuAssoc_tgt
  RecoMuonValidator recoMuonVMuAssoc_trk
  RPCDcsInfo rpcDcsInfo
  RPCEfficiency rpcEfficiency
  DQMEventInfo rpcEventInfo
  RPCFEDIntegrity rpcFEDIntegrity
  RPCRecHitValid rpcRecHitV
  RPCMonitorDigi rpcdigidqm
  RPCRecHitProbability rpcrechitprobability
  SiStripFEDCheckPlugin siStripFEDCheck
  SiStripFEDMonitorPlugin siStripFEDMonitor
  SingleTopTChannelLeptonDQM singleTopElectronMediumDQM
  SingleTopTChannelLeptonDQM_miniAOD singleTopElectronMediumDQM_miniAOD
  SingleTopTChannelLeptonDQM singleTopMuonMediumDQM
  SingleTopTChannelLeptonDQM_miniAOD singleTopMuonMediumDQM_miniAOD
  PFMuonDQMAnalyzer slimmedMuonValidation1
  SegmentTrackAnalyzer staMuonSegmentAnalyzer
  MuonTrackValidator staMuonTrackVMuonAssoc
  MuonTrackValidator staRefitMuonTrackVMuonAssoc
  MuonTrackValidator staRefitUpdMuonTrackVMuonAssoc
  MuonTrackValidator staSeedTrackVMuonAssoc
  MuonTrackValidator staUpdMuonTrackVMuonAssoc
  PhotonAnalyzer stdPhotonAnalysis
  SiStripDigiValid stripDigisValid
  SiStripRecHitsValid stripRecHitsValid
  RecoSusyDQM susyDQM
  TauValidation tauValidation
  TauValidationMiniAOD tauValidationMiniAODQCD
  TauValidationMiniAOD tauValidationMiniAODRealData
  TauValidationMiniAOD tauValidationMiniAODRealElectronsData
  TauValidationMiniAOD tauValidationMiniAODRealMuonsData
  TauValidationMiniAOD tauValidationMiniAODZEE
  TauValidationMiniAOD tauValidationMiniAODZMM
  TauValidationMiniAOD tauValidationMiniAODZTT
  MuonTrackValidator tevMuonDytTrackVMuonAssoc
  MuonTrackValidator tevMuonFirstTrackVMuonAssoc
  MuonTrackValidator tevMuonPickyTrackVMuonAssoc
  TkConvValidator tkConversionValidation
  TopSingleLeptonDQM topSingleElectronMediumDQM
  TopSingleLeptonDQM_miniAOD topSingleElectronMediumDQM_miniAOD
  TopSingleLeptonDQM topSingleMuonMediumDQM
  TopSingleLeptonDQM_miniAOD topSingleMuonMediumDQM_miniAOD
  TotemDAQTriggerDQMSource totemDAQTriggerDQMSource
  TotemRPDQMHarvester totemRPDQMHarvester
  TotemRPDQMSource totemRPDQMSource
  TrackerHitAnalyzer trackerSiStripHitsValid
  TrackingTruthValid trackingTruthValid
  TriggerMatchMonitor triggerMatchMonitor_miniAOD
  MuonTrackValidator trkMuonTrackVTrackAssoc
  MuonTrackValidator trkProbeTrackVMuonAssoc
  V0Validator v0Validator
  MuonSimHitsValidAnalyzer validSimHit
  RPCDigiValid validationMuonRPCDigis
  PrimaryVertexAnalyzer4PUSlimmed vertexAnalysis
  WValidation wMinusEleValidation
  WValidation wMinusMuoValidation
  WValidation wPlusEleValidation
  WValidation wPlusMuoValidation
  ZToMuMuGammaAnalyzer zmumugammaAnalysis
  ZToMuMuGammaAnalyzer zmumugammaOldAnalysis
  EcalSimHitsValidation ecalSimHitsValidation
  EcalBarrelSimHitsValidation ecalBarrelSimHitsValidation
  EcalEndcapSimHitsValidation ecalEndcapSimHitsValidation
  EcalPreshowerSimHitsValidation ecalPreshowerSimHitsValidation
```

**Comment by srimanob** [comment]
@schneiml @andrius-k Could you please have a look? Thx.

**Comment by schneiml** [comment]
@Dr15Jones yes, since the Per-Lumi saving project from May they all need to watch lumis. We could try to not do that anymore, but it only makes sense once we got rid of the modules with hard dependencies on lumis (and legacy modules etc.), which primarily requires massively trimming down the sequences we run by default.

So once we have a more minimal set of DQM sequences to run, we could move this WF to it and then have a chance to no longer have legacy modules. Then, we could consider disabling the per-lumi saving again and get concurrent lumis.

To get the per-lumi saving back, we need the new DQMStore project. There, we still have the explicitly lumi-watching one modules (~10), and legacy modules, so cleaning up these is needed either ways.

In general, a very much appreciate a workflow like this, testing lumi transitions is useful in many cases.
